src/bootstrap/dist.rs

```diff
@@ -1444,6 +1452,9 @@ impl Step for Extended {
             if rls_installer.is_some() {
                 prepare("rls");
             }
+            if rustfmt_installer.is_some() {
+                prepare("rustfmt");
```

Comment:
Have you tested this? I don't think it will work as the combined installers aren't ready for this 

(similar to other modifications below)